src/node.rs

```diff
@@ -422,18 +422,17 @@ impl<K, V> TreeBin<K, V> {
         // there are three cases when a bin pointer is invalidated:
         //
         //  1. if the table was resized, bin is a move entry, and the resize has completed. in
-        //     that case, the table (and all its heads) will be dropped in the next epoch
-        //     following that.
+        //     that case, the table (and all its heads) will be retired.
```

Comment:
same note as for the other instances of this list

Reply:
Fixed.